src/api/OracleAPI.ts

```diff
@@ -64,13 +64,13 @@ export class OracleAPI {
   }
 
   /**
-   * Returns the Unix timestamp of when price can feed can next be updated
+   * Returns the Unix timestamp of earliest time the TimeSeriesFeed can be updated
    *
-   * @param  historicalPriceFeed    Address of the HistoricalPriceFeedV2 contract to poll
+   * @param  timeSeriesFeed         Address of the TimeSeriesFeed contract to poll
    * @return                        Timestamp of when the price feed can be updated next
    */
-  public async getRollingHistoricalFeedNextAvailableUpdateAsync(historicalPriceFeed: Address): Promise<BigNumber> {
-    return await this.historicalPriceFeedV2Wrapper.nextAvailableUpdate(historicalPriceFeed);
+  public async getRollingHistoricalFeedNextEarliestUpdateAsync(timeSeriesFeed: Address): Promise<BigNumber> {
```

Comment:
I get that we're trying to keep the interface the same. But would it be worth it to rename to `getTimeSeriesFeedNextEarliestUpdateAsync`?

Reply:
The interface is already changed so I have no problem with that